riscv-asm.md

```diff
@@ -377,6 +377,49 @@ for PIC as seen by objdump:
 			4: R_RISCV_PCREL_LO12_I	.L0
 ```
 
+Load and Store Global
+---------------------
+
+The following pseudo instructions are available to load from and store to
+global objects:
+
+  * `l{b|h|w|d} <rd>, <symbol>`: load byte, half word, word or double word from global
+  * `s{b|h|w|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global
+  * `fl{w|d} <rd>, <symbol>, <rt>`: load float or double from global
+  * `fs{w|d} <rd>, <symbol>, <rt>`: store float or double to global
+
+The following example shows how these pseudo instructions are used:
+
+```assembly
+	lw	a0, var1
+	fld	fa0, var2, t0
+	sw	a0, var3, t0
+	fsd	fa0, var4, t0
+```
+
+Which generates the following assembler output and relocations
+as seen by `objdump`:
```

Comment:
This is specifically medany output which should perhaps be mentioned, as you will see different code with other code models.

Reply:
Again, are you sure that it's not the same in `medlow`?

Reply:
I checked gas, and see that you are right.  Though I'm tempted to call this a bug.  We normally shouldn't be using auipc in medlow code, except for calls where we always use auipc..  Though as I mentioned elsewhere, medany code can be linked into medlow code, and the result is medlow, so this is safe.  It is just unexpected.

Reply:
How do you specify the code model to `gas`?  Did I miss it in its man page?

Reply:
You can't specify the code model to gas.  Looks like an oversight, which is why the macros all emit medany code, because medany code works for both medlow and medany.  But that won't work for compact or FDPIC, which means those macros won't be usable with non-medlow/medany code models unless we change how this works.

Reply:
Or maybe won't work with other code models, depending on the amount of compatibility with the existing medlow/medany code models.